simple-blog/src/app/app.component.html

```diff
@@ -1,11 +1,14 @@
 <body>
   <!-- mat-toolbar needs to be refactored to be its own component so it can be reusable -->
   <mat-toolbar id="navbar" color="primary">
-    <div id="nav-title">Jose's Blog</div>
+    <div id="nav-title">
+      <a href="http://localhost:4200/">Jose's Blog</a>
```

Comment:
This should say Simple Blog, not Jose's Blog. I think you did not pull develop before you created your branch. Also this branch does not follow the branch naming convention I specified in the README.md. The branch naming convention is important, because it is what allows the pipeline to know when to kick off. However, I did create a new pr updating the pipeline. Let's merge that first, and then I suggest creating a new branching following the naming convention I suggest in the new updated README.md